tests/functional/adapter/materialization/test_view_security.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+from dbt.tests.util import run_dbt
+
+from tests.functional.adapter.materialization.fixtures import (
+    seed_csv,
+    model_sql,
+    profile_yml,
+)
+
+
+class TestViewSecurity:
+    """
+    Methods in this class will be of two types:
+    1. Fixtures defining the dbt "project" for this test case.
+       These are scoped to the class, and reused for all tests in the class.
+    2. Actual tests, whose names begin with 'test_'.
+       These define sequences of dbt commands and 'assert' statements.
+    """
+
+    # configuration in dbt_project.yml
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "name": "example",
+            "models": {
+                "+materialized": "view",
+                "+view_security": "invoker"
+            }
+        }
+
+    # everything that goes in the "seeds" directory
+    @pytest.fixture(scope="class")
+    def seeds(self):
+        return {
+            "seed.csv": seed_csv,
+        }
+
+    # everything that goes in the "models" directory
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {
+            "materialization.sql": model_sql,
+            "materialization.yml": profile_yml,
+        }
+
+    # The actual sequence of dbt commands and assertions
+    # pytest will take care of all "setup" + "teardown"
+    def test_run_seed_test(self, project):
+        """
+        Seed, then run, then test. We expect one of the tests to fail
+        An alternative pattern is to use pytest "xfail" (see below)
+        """
+        # seed seeds
+        results = run_dbt(["seed"])
+        assert len(results) == 1
+        # run models
+        results = run_dbt(["run"])
+        assert len(results) == 1
```

Comment:
please check whether the command has been executed successfully.

https://docs.getdbt.com/docs/contributing/testing-a-new-adapter

```
        results = run_dbt(["test"], expect_pass = False) # expect failing test
        assert len(results) == 2
        # validate that the results include one pass and one failure
        result_statuses = sorted(r.status for r in results)
        assert result_statuses == ["fail", "pass"]
```

Reply:
I added `expect_pass = True`